src/components/pageheader/PageHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './pageheader.less';
+import { Typography } from 'antd';
+const { Title, Paragraph } = Typography;
+
+type PageHeaderProps = {
+  readonly pageTitle: string;
+  readonly pageSubtitle: string;
+  readonly subtitleColor: string;
+};
+
+const PageHeader: React.FC<PageHeaderProps> = ({
+  pageTitle,
+  pageSubtitle,
+  subtitleColor,
+}) => {
+  return (
+    <div className="header">
+      <Title id="title">{pageTitle}</Title>
```

Comment:
Why can't className be used for this?

Reply:
Using Title, I wasn't able to style certain things with className but I've fixed that with !important (same with Paragraph and subtitleColor above).

Reply:
Unfortunately, using `!important` can cause a lot of unintended side effects, especially when we're working with the Ant.D library -`!important` has the potential to override AntD's internal styles in unpredictable ways. 

A good way to properly overwrite the styles without using `!important` is using styled components. There is some WIP documentation in https://github.com/Code-4-Community/c4c-dev-docs/pull/12/files. 

Ultimately, you'll have something like: 

```
const StyledTitle = styled(Title)`
   your styles
`;

<StyledTitle>{pageTitle}</StyledTitle>
```

I believe the dependencies are not installed currently, you can install them following the guide in the docs. Let me know if you have any questions on this :) 

cc @rymaju